iron-flex-layout-classes.html

```diff
@@ -370,7 +370,6 @@
         display: block;
       }
 
-      /* IE 10 support for HTML5 hidden attr */
       [hidden] {
```

Comment:
i think it's still useful -- remember, `[hidden]` is a lie, and if you don't define this it will never be applied correctly once a `display:xxx` has been set anywhere else